lib/Cake/Model/Datasource/CakeSession.php

```diff
@@ -527,6 +537,10 @@ protected static function _configureSession() {
 		if (isset($sessionConfig['timeout']) && !isset($sessionConfig['cookieTimeout'])) {
 			$sessionConfig['cookieTimeout'] = $sessionConfig['timeout'];
 		}
+		if (isset($sessionConfig['useForwardsCompatibleTimeout']) && $sessionConfig['useForwardsCompatibleTimeout']) {
+				static::$_useForwardsCompatibleTimeout = true;
```

Comment:
This looks like it has an extra layer of indentation.

Reply:
I'll have a look

Reply:
You were right, you hawk :-)